libs/design/src/atoms/button/button.component.scss

```diff
@@ -89,6 +89,7 @@
 .daff-underline-button {
 	@include daff-button();
 	background: transparent;
+	border: 0;
```

Comment:
I think this is necessary for `button[daff-underline-button]

Reply:
Yes, that's why I added it.